<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Text Analyser</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Determine Language</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="StaticLang">
            <property name="toolTip">
             <string>Set if the languages are static.</string>
            </property>
            <property name="whatsThis">
             <string>When the languages are static and when the content of a document is changed, the system supposes that the language of this document has not changed.&lt;br&gt;
When the languages are not static, the language of a document is always determinewhen the document is analysed.</string>
            </property>
            <property name="text">
             <string>Static Languages</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>Minimum stopword ratio</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="KDoubleNumInput" name="MinStopWords">
              <property name="toolTip">
               <string>Set the minimum stopword ratio.</string>
              </property>
              <property name="whatsThis">
               <string>To determine the language of a document, the following method is used:
&lt;ol&gt;
&lt;li&gt; For each language defined in the system, the ratio of the number of different stopwords over the overall number of different words contained in the document is computed.&lt;/li&gt;
&lt;li&gt; All the languages having this ratio lower than the &lt;b&gt;minimum stopword ratio&lt;/b&gt; are elminated.&lt;/li&gt;
&lt;li&gt; The language with the highest number of occurences of stopwords is assigned to the document.&lt;li&gt;
&lt;/ol&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Valid Words</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="NonLetterWords">
            <property name="toolTip">
             <string>Should non letter words be used.</string>
            </property>
            <property name="whatsThis">
             <string>When analysing a document, it is possible to use for the description only words containing letters.</string>
            </property>
            <property name="text">
             <string>Non letter words are used</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel2">
              <property name="text">
               <string>Minimum size of a word</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="KIntNumInput" name="MinWordSize">
              <property name="toolTip">
               <string>Specify the minimum size of words.</string>
              </property>
              <property name="whatsThis">
               <string>When a word is contained in the document and its size is lower than this threshold, the word is not used.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupFiltering">
         <property name="title">
          <string>Filtering</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="Filtering">
            <property name="text">
             <string>Use Word Filtering?</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel1_2">
              <property name="toolTip">
               <string>Maximum number of following occurences of same letter .</string>
              </property>
              <property name="whatsThis">
               <string>Maximum number of following occurences of same letter.</string>
              </property>
              <property name="text">
               <string>Maximal occurences</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="KIntNumInput" name="NbSameOccur">
              <property name="toolTip">
               <string>of a same word in a row to consider a word non-valid</string>
              </property>
              <property name="whatsThis">
               <string>of a same word in a row to consider a word non-valid</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel2_2">
              <property name="toolTip">
               <string>Value of the ratio of normal letters on the total number of letters to be considered as a valid.</string>
              </property>
              <property name="whatsThis">
               <string>Value of the ratio of normal letters on the total number of letters to be considered as a valid.</string>
              </property>
              <property name="text">
               <string>Normal Ratio</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="KDoubleNumInput" name="NormalRatio"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="spacer11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>XML Structure</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox7">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="UseDefaultNamespace">
              <property name="text">
               <string>Use default namespace</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="DefaultNamespace"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="ExtractStruct">
            <property name="text">
             <string>Extract tags and attributes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="FullIndex">
            <property name="text">
             <string>Full index (structure and positions)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox6">
         <property name="title">
          <string>Elements Considered As Content</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="StructIsContent">
              <property name="text">
               <string>Structurral tokens</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer28">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="KDoubleNumInput" name="WeightStruct"/>
            </item>
            <item>
             <spacer name="spacer15_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="ExtractValues">
              <property name="text">
               <string>Attributes values</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer29">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>30</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="KDoubleNumInput" name="WeightValues"/>
            </item>
            <item>
             <spacer name="spacer16">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>16</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Metadata Tags</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="ExtractIndex">
         <property name="text">
          <string>Extract declarative tags?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="DetectMetaTag">
         <property name="text">
          <string>Detect Tags</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="KListWidget" name="IndexTags"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="AddTag">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveTag">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="ChildTags">
         <property name="text">
          <string>Declarative tags may have child tags?</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_2">
         <item>
          <widget class="QLabel" name="textLabel1_3">
           <property name="text">
            <string>Maximum terms in a declarative tag</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KIntNumInput" name="MaxTerms">
           <property name="minValue" stdset="0">
            <number>0</number>
           </property>
           <property name="maxValue" stdset="0">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_3">
         <item>
          <widget class="QLabel" name="textLabel2_3">
           <property name="text">
            <string>Maximum depth of a declarative tag</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KIntNumInput" name="MaxDepth">
           <property name="minValue" stdset="0">
            <number>0</number>
           </property>
           <property name="maxValue" stdset="0">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_4">
         <item>
          <widget class="QLabel" name="textLabel1_5">
           <property name="text">
            <string>Maximal number of occurences</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KIntNumInput" name="MaxOccurs"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>Filtering</sender>
   <signal>toggled(bool)</signal>
   <receiver>NbSameOccur</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Filtering</sender>
   <signal>toggled(bool)</signal>
   <receiver>NormalRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractStruct</sender>
   <signal>toggled(bool)</signal>
   <receiver>StructIsContent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StructIsContent</sender>
   <signal>toggled(bool)</signal>
   <receiver>WeightStruct</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractValues</sender>
   <signal>toggled(bool)</signal>
   <receiver>WeightValues</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseDefaultNamespace</sender>
   <signal>toggled(bool)</signal>
   <receiver>DefaultNamespace</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleStruct(bool)</slot>
  <slot>toggleFullIndex(bool)</slot>
 </slots>
</ui>
